browser/brave_stats_updater.cc

```diff
@@ -68,11 +72,19 @@ GURL GetUpdateURL(const GURL& base_update_url,
 
 namespace brave {
 
-std::string BraveStatsUpdater::g_base_update_url_(
-    "https://laptop-updates.brave.com/1/usage/brave-core");
-
 BraveStatsUpdater::BraveStatsUpdater(PrefService* pref_service)
-    : pref_service_(pref_service) {}
+    :  threshold_score_(0),
+       threshold_ping_state_(THRESHOLD_PING_INACTIVE),
+       pref_service_(pref_service) {
+  std::unique_ptr<base::Environment> env(base::Environment::Create());
```

Comment:
use command line flags to override instead of env and pass the full url. I think you only need one flag to override the url and not two